src/scripts.js

```diff
@@ -78,20 +78,19 @@ function displayFavoritesView() {
   domUpdates.hide(domUpdates.allRecipesSection);
   domUpdates.hide(domUpdates.weeklyRecipesSection);
   domUpdates.show(domUpdates.favoriteRecipesSection);
-
+  clearTags();
   checkFavoriteRecipes();
 }
 
 function checkFavoriteRecipes() {
```

Comment:
Since the tag selection(s) do not persist when they leave the favorite's view, checking to see if there are tags selected is no longer necessary and the only condition to check is whether the user has favorites.